tests/testthat/setup-fakedata.R

```diff
@@ -152,3 +152,33 @@ export_fact <- function() {
 }
 
 tt_to_export <- export_fact()
+
+# Creating data-set with wide content to test wrapping
+tt_to_test_wrapping <- function() {
+    trimmed_data <- ex_adsl %>% 
+        filter(SEX %in% c("M", "F")) %>% 
+        filter(RACE %in% levels(RACE)[1:2]) 
+    
+    levels(trimmed_data$ARM)[1] <- "Incredibly long column name to be wrapped"
+    levels(trimmed_data$ARM)[2] <- "This_should_be_somewhere_split"
+    
+    basic_table(title = "Enough long title to be probably wider than expected",
+                main_footer = "Also this seems quite wider than expected initially.") %>%
+        split_cols_by("ARM") %>%
+        split_rows_by("RACE", split_fun = drop_split_levels) %>%
+        analyze(c("AGE", "EOSDY"), 
+                na_str = "A very long content to_be_wrapped_and_splitted",
+                inclNAs = TRUE) %>%
+        build_table(trimmed_data)
+}
+
+tt_for_wrap <- tt_to_test_wrapping()
+
+# Helper function in R base to count how many times a pattern appears in a string
```

Comment:
This only works if pattern is a single character, please change the name of it and document that fact in a comment, in case we ever want to use it elsewhere

Reply:
Done. Let me know if it is fine as it is now ;)